### ISSUE TITLE
Do you train with TRADES and ALP or one of them?

### ISSUE THREAD

**Issue by headscott** [FPNAS/LBGAT#11]
When I run

`python train_lbgat_cifar100.py -mark cifar100_lbgat0_0.031 -teacher_model ResNet18_cifar --beta 0`

will the model be trained with ALP or TRADES? If not, how can I activate it?

**Comment by jiequancui** [comment]
Thanks for your interest in our work.
For the TRADES baseline, we use their official code. Please refer to https://github.com/yaodongyu/TRADES.

**Comment by headscott** [comment]
Yeah, I know TRADES, but is this method also used, when I run the mentioned command? Or do I need special parameters?

**Comment by headscott** [comment]
And I saw in your paper, that you the Teacher Model is trained too, but I don't see, that the teacher starts its train mode by calling:

`model_teacher.train()`

I mean, there is no call of this type. So is the model_teacher pre-trained then?
